components/button/button-icon.js

```diff
@@ -20,17 +20,18 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 			/**
 			 * Aligns the leading edge of text if value is set to "text"
 			 * @type {'text'|''}
-			 * @default ""
```

Comment:
`@default` doesn't seem to work. It more just looks at if there's an initial value set in the constructor. This case is weird anyways since the default isn't really `""`, it's more undefined.